script/Run.sc

```diff
@@ -1,121 +1,31 @@
 import sys.process._
 import ammonite.ops._
-import java.io.File
 
 object Job extends Enumeration {
   type Job = Value
-  val Index, Deploy, Elastic, Test = Value
+  val Index, Elastic = Value
 }
 
 implicit val readCiType: scopt.Read[Job.Value] = scopt.Read.reads(Job withName _)
 
-def ensureSuccess(status: Int): Unit =
+def run(args: String*): Unit = {
+  val status = Process(args.toList).!
   if (status == 0) ()
-  else sys.error(s"Process exited with status $status")
-
-def run(args: String*): Unit = ensureSuccess(Process(args.toList).!)
-def runD(args: String*)(dir: Path): Unit = {
-  println(s"Running command ${args.mkString(" ")} in working directory $dir")
-  ensureSuccess(Process(args.toList, Some(dir.toIO)).!)
-}
-def runSlurp(args: String*): String = Process(args.toList).lineStream.toList.headOption.getOrElse("")
-def runPipe(args: String*)(file: Path) = ensureSuccess((Process(args.toList) #> file.toIO).!)
-def runEnv(args: String*)(envs: (String, String)*) = {
-  println(args.toList)
-  println(envs)
-  ensureSuccess(Process(command = args.toList, cwd = None, extraEnv = envs: _*).!)
-}
-
-def sbt(commands: String*): Unit = {
-  val jvmOpts =
-    "-Dsbt.global.staging=/tmp" ::
-    "-DELASTICSEARCH=remote" ::
-    "-Xms1G" ::
-    "-Xmx3G" ::
-    Nil
-
-  // run index
-  runEnv("./sbt", ("clean" :: commands.toList).mkString(";", " ;", ""))(("JVM_OPTS", jvmOpts.mkString(" ")))
-}
-
-def datetime = {
-  import java.text.SimpleDateFormat
-  import java.util.Calendar
-  new SimpleDateFormat("yyyy/MM/dd HH:mm:ss").format(Calendar.getInstance().getTime()) 
-}
-
-def updatingRepositories(contribPath: Path, indexPath: Path)(f: () => Unit): Unit = {
-  println("Pulling the latest data from the 'contrib' repository")
-  runD("git", "checkout", "master")(contribPath)
-  runD("git", "remote", "update")(contribPath)
-  runD("git", "pull", "origin", "master")(contribPath)
-
-  // run index
-  f()
-
-  println("Pushing current state to the 'index' repository")
-  runD("git", "add", "-A")(indexPath)
-  runD("git", "commit", "--allow-empty", "-m", '"' + datetime + '"')(indexPath)
-  runD("git", "push", "origin", "master")(indexPath)
+  else sys.error(s"Process '${args.mkString(" ")}' exited with status $status")
 }
 
 /**
-  * @param reposDir Directory that contains the `scaladex-credentials`, `scaladex-index` and `scaladex-contrib`
-  *                 sub-directories containing the git repositories
+  * @param dataPath Path of the deployed `data` application
+  * @param reposDir Directory that contains the `scaladex-credentials`, `scaladex-index` and
+  *                 `scaladex-contrib` sub-directories containing the git repositories
   */
-@main def main(reposDir: String, fullBranchName: String, job: Job.Value) = {
+@main def main(dataPath: String, reposDir: String, job: Job.Value) = {
```

Comment:
This script is useful to trigger a re-indexing or to re-create the ElasticSearch database.